PetaPoco.Tests.Integration/Databases/OracleTests/OracleStoredProcTests.cs

```diff
@@ -14,5 +20,193 @@ public OracleStoredProcTests()
             : base(new OracleTestProvider())
         {
         }
+
+        private IDataParameter GetOutputParameter() => new OracleParameter("p_out_cursor", OracleDbType.RefCursor, ParameterDirection.Output);
```

Comment:
More of a question here, use your judgment, but would using a param with default value be a good idea here, so it's clear the out RefCursor isn't locked in to that particular name for future tests? e.g.:

```cs
private IDataParameter GetOutputParameter(string parameterName = "p_out_cursor")
	=> new OracleParameter(parameterName, OracleDbType.RefCursor, ParameterDirection.Output);
```

Reply:
For simplicity and consistency's sake I opted for encapsulating the name,
since it has to match the name used in the stored procedure.
I initially called the method GetOutputCursor, but then I saw
GetDataParameter method and thought GetOutputParameter would fit nicely.
Thinking about it now, GetCursorParameter also fits pretty well. All sides
same side I suppose. Naming is so difficult lol.